newVersion.py

```diff
@@ -97,21 +115,35 @@ def build_dir(dir_name):
     def __iter__(self):
         return iter(self.dirs)
 
+def is_itwin_non_core_project_line(line, is_itwin):
+    if not is_itwin: return False
+    for itwin_non_core_package in itwin_non_core_packages:
+        if line.find(f'@itwin/{itwin_non_core_package}') != -1:
+            return True
+    return False
+
+def parse_replacement_tuple(tuple):
+    if len(tuple) == 2:
+        return (tuple[0], tuple[1], False)
+    else:
+        return (tuple[0], tuple[1], tuple[2])
```

Comment:
I think the following should work:

```suggestion
def parse_replacement_tuple(tuple):
    return (tuple[0], tuple[1], tuple[2] if len(tuple) == 2 else False)
```

Reply:
It doesn't work when I try it (seems to just crash?). It might be possible to do something like that, but the difficulty of testing means that I don't think it's worth it.

Reply:
OK, thanks for trying.